common/rpc/interceptor/telemetry.go

```diff
@@ -158,7 +159,7 @@ func (ti *TelemetryInterceptor) emitActionMetric(
 	metricsHandler metrics.MetricsHandler,
 	result interface{},
 ) {
-	if _, ok := grpcActions[methodName]; !ok || !strings.Contains(strings.ToLower(fullName), strings.ToLower(common.FrontendServiceName)) {
+	if _, ok := grpcActions[methodName]; !ok || !strings.Contains(strings.ToLower(fullName), strings.ToLower(frontendServicePackageName)) {
```

Comment:
instead of check contains, please check HasPrefix. And check the prefix against 
workflowserviceGrpcPrefix = "/temporal.api.workflowservice.v1.WorkflowService/"